src/Form/matching.js

```diff
@@ -415,7 +441,6 @@ class Matching {
         for (let stringName of matchableStrings) {
             let elementString = this.activeElementStrings[stringName]
             if (!elementString) continue
-            elementString = elementString.toLowerCase()
```

Comment:
I've run a whole bunch of performance tests and lowercasing was never more performant than using the `i` flag on the regex (it is marginally slower). It's largely irrelevant because this is not the bottleneck anyway, so I've removed all the lowercasing.